jmix-rest/rest/src/main/java/io/jmix/rest/impl/service/filter/RestFilterParser.java

```diff
@@ -227,21 +218,45 @@ protected RestFilterPropertyCondition parsePropertyCondition(JsonObject conditio
             } else {
                 value = parseValue(metaProperty, valueJsonElem.getAsString());
             }
-            condition.setValue(transformValue(value, op));
-            condition.setQueryParamName(generateQueryParamName());
+            condition.setParameterValue(value);
+        } else {
+            if ("isNull".equals(operator)) condition.setParameterValue(false);
```

Comment:
Avoid literals. Perhaps create a class with string constants for all supported REST operators. Or an enum with a second field - corresponding Condition operation.